cmd/snap/cmd_get.go

```diff
@@ -252,8 +256,11 @@ func (x *cmdGet) Execute(args []string) error {
 
 	var conf map[string]interface{}
 	var err error
-	// TODO: check that the experimental aspect feature is enabled
 	if isAspectID(snapName) {
+		if err := validateAspectFeatureFlag(); err != nil {
```

Comment:
this is just an optimization not to call snapd, right?

Reply:
Yes, it's already gated at the API but we get this for free since there's already code to check the flag and add the aspect-related documentation depending on it